### ISSUE TITLE
Implement ops

### ISSUE THREAD

**Issue by rom1504** [mhsjlw/flying-squid#41]
Only ops should be able to kick and ban.

**Comment by mhsjlw** [comment]
We should first implement a permissions system.

**Comment by rom1504** [comment]
Well this is the permissions system

**Comment by mhsjlw** [comment]
Yes, but I mean not like normal minecraft's permission system. I am talking about a *good* one

**Comment by mhsjlw** [comment]
Something like, (not real JSON, I think)

```json
[
{
  "name": "mhsjlw",
  "permissions": [
    "*"
  ]
},
{
  "name": "rom1504",
  "permissions": [
    "world-guard.*"
  ]
}
]
```

**Comment by demipixel** [comment]
I think it would be best to simply implement ops (wouldn't be hard, just player.op) and use that ("return" inside a command if the player is not an op). I imagine somebody will make an advanced permissions base-external-plugin that other plugins will build on top of.

Just my personal opinion so we can get as similar to vanilla as possible.

**Comment by rom1504** [comment]
I'm fine with both solutions.

**Comment by demipixel** [comment]
Sorta done in the other PR

**Comment by rom1504** [comment]
We're still missing the saving/loading part

**Comment by demipixel** [comment]
Okay, although that seems like a completely seperate issue considering nothing gets saved yet :P

**Comment by rom1504** [comment]
well, ops saving/loading is fairly independent from all the rest of the saving if we follow vanilla format (it's just a ops.json file) (https://github.com/rom1504/node-minecraft-wrap/blob/master/lib/wrap.js#L113)
